packages/fxa-auth-server/lib/senders/emails/templates/cadReminderFirst/index.stories.ts

```diff
@@ -44,16 +38,8 @@ const commonPropsWithOverrides = (
     },
   });
 
-export const CadReminderDesktopTablet = Template.bind({});
-CadReminderDesktopTablet.args = commonPropsWithOverrides({
-  onDesktopOrTabletDevice: true,
-});
-CadReminderDesktopTablet.storyName = 'User is on desktop or tablet device';
-
```

Comment:
@LZoog I removed the desktop state from both the cadReminders. I think we do not want to show the desktop version to the user and since we're also not passing the `onDesktopOrTabletDevice` from [`email.js`](https://github.com/mozilla/fxa/blob/949a3d73e56832bf855fdec18f9b55901fa65acd/packages/fxa-auth-server/lib/senders/email.js#L2902) file, it made sense to remove that state entirely from storybook as well